spec/presenters/tree_node_builder_spec.rb

```diff
@@ -338,4 +343,46 @@
       node[:expand].should eq(true)
     end
   end
+
+  context "#node_with_display_name" do
+    before do
+      @root_tenant = EvmSpecHelper.create_root_tenant
+    end
+    it "should return node text with Disabled in the text for Disabled domain" do
+      domain = FactoryGirl.create(:miq_ae_domain,
```

Comment:
@h-kataria 
The miq_ae_domain factory sets the default tenant as root_tenant. You should be able to change this to
domain = FactoryGirl.create(:miq_ae_domain, :name => "test1", enabled => false)
You don't need the parent => nil